src/styles/blocks/cards/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background-color: $card-bg-color;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  transition: all 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__photo {
+    display: flex;
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $font-color-main;
+    margin-bottom: 4px;
+
+    .card:hover & {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $font-color-secondary;
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+  }
+
+  &__reviews {
+    display: flex;
+    align-self: flex-end;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $font-color-main;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-text {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $font-color-secondary;
+    }
+  
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__link {
+    display: flex;
+    height: 40px;
+    width: 166px;
+    border-radius: 5px;
+    box-sizing: border-box;
+    align-items: center;
+    justify-content: center;
+    color: $card-bg-color;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    background-color: $button-color-main;
+    border: 1px solid $border-color;
+    transition: all 0.5s;
+
+    &:hover {
+      color: $button-color-main;
+      background-color: $card-bg-color;
+      border: 1px solid $button-color-main;
+    }
+    
+    .card:hover & {
+      color: #00acdc;
+      background-color: #fff;
+    }

```

Comment:
Can you clarify, for what purpose did you add these styles?

Reply:
> Can you clarify, for what purpose did you add these styles?

I do not remember exactly. I'm pretty sure it was some sort of experiment that I forgot to delete.